conferences/abstracts/UKRI-cloud-workshop-abstract.md

```diff
@@ -17,3 +17,7 @@ By harnessing the power of cloud computing and building a BinderHub with expande
 We can improve reproducibility by making it easy to showcase and re-run analyses, improve reliability by making it easy to review analyses and catch bugs early in the development process without installation, and improve the reusability of the code by capturing the computing environment.
 
 [1] Project Jupyter, Bussonnier, M., Forde, J., Freeman, J., Granger, B., Head, T., … Willing, C. (2018). Binder 2.0 - Reproducible, interactive, sharable environments for science at scale. In Proceedings of the 17th Python in Science Conference. SciPy. https://doi.org/10.25080/majora-4af1f417-011
+
+## Submission Info
+
+This was submitted as a Demonstration under the "Cloud Hosting for Virtual Research Environments" category.
```

Comment:
Thanks - can you also add the links to the application form and any other information about the conference? Maybe up at the top? Or down here, up to you 😃

Reply:
Since the application is on eventbrite, would the link still work after applications have closed or is it a permalink?

I've included a general link to the Cloud Working Group site for now, I'll update it to be the workshop specific one when that's live, but I guess that will be after the conference.